.github/workflows/acceptance-tests.yml

```diff
@@ -12,7 +12,53 @@ on:
   pull_request: # you can run a specic job in your PR using GitHub labels
     types: [ labeled ]
 
+env:
+    FORCE_TEST_ACC: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || inputs.parent-event-name == 'release' }}
+    TF_LOG: ${{ vars.LOG_LEVEL }}
+    SKIP_TEST_EXTERNAL_CREDENTIALS: ${{ vars.SKIP_TEST_EXTERNAL_CREDENTIALS }}
+    ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
+    PARALLEL_GO_TEST: "20"
```

Comment:
are these "global" env?

Reply:
they are passed to all jobs

Reply:
are we sure this is a good idea? Here is where I would like to see all environment variables as close as possible to the job/step so I don't have to go up and down in the file to see the context.

Reply:
we can decide what we want to have closer and which one we want for all. 
for instances in the beginning I was going to delete PARALLEL_GO_TEST and ACCTEST_TIMEOUT because they have their default values, but in this way you can change the var instead of having to change code.

about FORCE_TEST_ACC I think it's a good idea to have it here as this depends on a specific test. For example we want to run tests always if it's a release

Reply:
i don't see a big problem in having common vars here. We can move them later if we realize that we want different values for different tests